docs/association-financials.md

```diff
@@ -0,0 +1,29 @@
+---
+outline: deep
+---
+
+# Wallets and Contracts controled by the association
+
+## Station Rewards
+
+::: tip 55,000,000 tokens vesting linearly for 10 years.
```

Comment:
Entering this page for the first time, I don't understand where I am, what I'm reading. Feels like it's missing a short intro. Also, jumping directly to this "tip", again feels out of place. Especially in a page called Financials, with a title "wallets and contracts..." and a header "station rewards". Seems sloppy.

Reply:
I think it's better now. Please check.